content/python/concepts/built-in-functions/terms/help/help.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'help()'
+Description: 'Display the documentation of the object passed as argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `help()` function accepts an object as a parameter, which is a module, function, class or keyword. After being called, the documentation of the requested object is displayed.
+If no argument is passed the interactive help utility starts up on the console.
+
+## Syntax
+
+The object is passed to the `help()` function as parameter:
+
+```pseudo
+help(object)
+```
+
+To start the interactive help utility, the function is called without argument:
+
+```pseudo
+help()
+```
+  
+## Example
+
+Below is an example where a build-in function is passed as argument:
+
+```py
+help(print)
+```
+Which produces the following output:
+```shell
+print(...)
+   print(value, ..., sep=' ', end='\n', file=sys.stdout, flush=False)
+   
+   Prints the values to a stream, or to sys.stdout by default.
+   Optional keyword arguments:
+   file:  a file-like object (stream); defaults to the current sys.stdout.
+   sep:   string inserted between values, default a space.
+   end:   string appended after the last value, default a newline.
+   flush: whether to forcibly flush the stream.
+```
+
+If no argument is present:
+
+```py
+help()
+```
+Which returns as output:
```

Comment:
There need to be line breaks before and after code blocks.

Reply:
@alenn97 after applying these line breaks, I would then suggest we switch the two examples in this "Example" section. So, the first example should be `help()` + start screen, followed by `help(print)` + documentation results.